packages/animations/pubspec.yaml

```diff
@@ -18,13 +18,11 @@ dev_dependencies:
   vector_math: ^2.1.0
 
 screenshots:
-  - description: 'Thumbnail of a container transform animation.'
-    path: example/demo_gifs/container_transform_thumbnail.webp
   - description: 'Examples of the container transform pattern.'
-    path: example/demo_gifs/container_transform_lineup.gif
+    path: example/screenshots/container_transform_lineup.webp
```

Comment:
Could we actually update the README.md file to use the webp images as well instead of the gifs and then delete the gifs from the repository? Seems odd to have the animations duplicated.

(We'd need to double-check that github support webp embeddings in README's so they show up when you view the readme here: https://github.com/flutter/packages/tree/main/packages/animations)

(And same question for pub: Is pub OK to show embedded webp's here: https://pub.dev/packages/animations)

Reply:
I'd be happy to do this in a follow-up after double-checking github etc. Would that be okay?

Reply:
Sounds good.